app/src/main/java/com/example/sherryuan/greenthumb/persistence/GreenThumbDatabase.kt

```diff
@@ -0,0 +1,30 @@
+package com.example.sherryuan.greenthumb.persistence
+
+import android.arch.persistence.room.Database
+import android.arch.persistence.room.Room
+import android.arch.persistence.room.RoomDatabase
+import android.arch.persistence.room.TypeConverters
+import android.content.Context
+
+/**
+ * Created by sherryuan on 2018-08-29.
+ */
+
+@Database(entities = [(Plant::class)], version = 1)
+@TypeConverters(Converters::class)
+abstract class GreenThumbDatabase : RoomDatabase() {
+    abstract fun plantDao(): PlantDao
+
+    companion object {
```

Comment:
What's a `companion object`?

Reply:
Kotlin doesn't have a `static` keyword so if you want something to belong to the class rather than instances of the class you have to put it in a companion object